cms/repository/postgresrepo.py

```diff
@@ -174,3 +174,102 @@ def create_case(self, title,purpose,description, contact_details, contact_addres
         session.commit()
         return new_case.case_id
 
+    def search_member(self, search_input):
```

Comment:
This file starts growing . we should organise into small files like we did for domain use cases